README.md

```diff
@@ -10,13 +10,19 @@
 
 This script works with python3.
 
-Pre-requisites: `python`, `pip`, `git`, `bazel`, `virtualenv` (strongly
+Pre-requisites: `python`, `pip`, `git`, `bazel`, [`venv`](https://docs.python.org/3/library/venv.html) (strongly
 recommended).
 
-Before you do anything, install the dependencies:
-
 ```
-# pip or pip3, depends on your setup.
+# Clone bazel-bench.
+$ git clone https://github.com/bazelbuild/bazel-bench.git
+$ cd bazel-bench
+
+# Create and activate the virtual environment.
```

Comment:
I never did this? What value does it add?

Reply:
It keeps the bazel-bench deps contained within this virtual environment and doesn't pollute your global env.

Reply:
Let's add then that it's an optional, but recommended step?

Reply:
Done.